### ISSUE TITLE
Missing onError handling while dealing with java.io.FileNotFoundException

### ISSUE THREAD

**Issue by Tadas44** [miguelbcr/RxPaparazzo#57]
RxPaparazzo version `0.4.3`

```
Fatal Exception: java.lang.IllegalStateException: Exception thrown on Scheduler.Worker thread. Add `onError` handling.
       at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:57)
       at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:422)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:152)
       at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:265)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
Caused by rx.exceptions.OnErrorNotImplementedException: Failed opening content provider: content://media/external/images/media/1367
       at rx.internal.util.InternalObservableUtils$ErrorNotImplementedAction.call(InternalObservableUtils.java:386)
       at rx.internal.util.InternalObservableUtils$ErrorNotImplementedAction.call(InternalObservableUtils.java:383)
       at rx.internal.util.ActionSubscriber.onError(ActionSubscriber.java:44)
       at rx.observers.SafeSubscriber._onError(SafeSubscriber.java:153)
       at rx.observers.SafeSubscriber.onError(SafeSubscriber.java:115)
       at rx.internal.operators.OnSubscribeDoOnEach$DoOnEachSubscriber.onError(OnSubscribeDoOnEach.java:87)
       at rx.internal.operators.OnSubscribeMap$MapSubscriber.onError(OnSubscribeMap.java:88)
       at rx.internal.operators.OperatorSubscribeOn$1$1.onError(OperatorSubscribeOn.java:59)
       at rx.internal.operators.OperatorSubscribeOn$1$1.onError(OperatorSubscribeOn.java:59)
       at com.miguelbcr.ui.rx_paparazzo.interactors.DownloadImage$1.call(DownloadImage.java:61)
       at com.miguelbcr.ui.rx_paparazzo.interactors.DownloadImage$1.call(DownloadImage.java:50)
       at rx.Observable.unsafeSubscribe(Observable.java:10140)
       at rx.internal.operators.OperatorSubscribeOn$1.call(OperatorSubscribeOn.java:94)
       at rx.internal.schedulers.CachedThreadScheduler$EventLoopWorker$1.call(CachedThreadScheduler.java:228)
       at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:55)
       at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:422)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:152)
       at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:265)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
Caused by java.io.FileNotFoundException: Failed opening content provider: content://media/external/images/media/1367
       at android.content.ContentResolver.openTypedAssetFileDescriptor(ContentResolver.java:1102)
       at android.content.ContentResolver.openAssetFileDescriptor(ContentResolver.java:904)
       at android.content.ContentResolver.openInputStream(ContentResolver.java:629)
       at com.miguelbcr.ui.rx_paparazzo.interactors.DownloadImage.getContent(DownloadImage.java:80)
       at com.miguelbcr.ui.rx_paparazzo.interactors.DownloadImage.access$100(DownloadImage.java:30)
       at com.miguelbcr.ui.rx_paparazzo.interactors.DownloadImage$1.call(DownloadImage.java:54)
       at com.miguelbcr.ui.rx_paparazzo.interactors.DownloadImage$1.call(DownloadImage.java:50)
       at rx.Observable.unsafeSubscribe(Observable.java:10140)
       at rx.internal.operators.OperatorSubscribeOn$1.call(OperatorSubscribeOn.java:94)
       at rx.internal.schedulers.CachedThreadScheduler$EventLoopWorker$1.call(CachedThreadScheduler.java:228)
       at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:55)
       at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:422)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:152)
       at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:265)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
```

**Comment by miguelbcr** [comment]
Hi @Tadas44, 
Thanks for the report!

I've released version [0.4.4](https://github.com/miguelbcr/RxPaparazzo/releases/tag/0.4.4) and [0.4.4-2x](https://github.com/miguelbcr/RxPaparazzo/releases/tag/0.4.4-2.x)

Let me know if it works for you.

**Comment by Tadas44** [comment]
Thanks, I'll let you know if same issue occurs
